Metaverse/Source/Metaverse/Public/Components/AttributeComponent.h

```diff
@@ -0,0 +1,38 @@
+// Copyright 2022, Cronos Labs. All Rights Reserved
+
+#pragma once
+
+#include "CoreMinimal.h"
+#include "Components/ActorComponent.h"
+#include "AttributeComponent.generated.h"
+
+
+UCLASS( ClassGroup=(Custom), meta=(BlueprintSpawnableComponent) )
+class METAVERSE_API UAttributeComponent : public UActorComponent
+{
+	GENERATED_BODY()
+
+public:
+	// Sets default values for this component's properties
+	UAttributeComponent();
+    // Called every frame
+	virtual void TickComponent(float DeltaTime, ELevelTick TickType, FActorComponentTickFunction *ThisTickFunction) override;
+	void SetBalance(float value);
```

Comment:
this function also be annotated with BLUEPRINT,
so that it can be used also in blueprint

Reply:
Yes, most of functions can be exported as blueprint, but that is too much, we can export it in the future if really needed.